cmd/dep/testdata/harness_tests/init/manifest-exists/final/manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
My question was more about the wierd symbol in the Files Changed area.  Likely a "no ending newline" meaning.